src/build123d/topology.py

```diff
@@ -9248,6 +9068,242 @@ def unwrap_topods_compound(
     return compound
 
 
+def get_top_level_topods_shapes(topods_shape: TopoDS_Shape) -> ShapeList[TopoDS_Shape]:
```

Comment:
the `ShapeList[T]` parameter `T` is bound to `Shape | Vector` so that `TopoDS_Shape` looks sketchy

Reply:
Removed the ShapeList, it isn't needed.